roles/openshift_cli/defaults/main.yml

```diff
@@ -1,2 +1,2 @@
 ---
-openshift_version: "{{ openshift_image_tag | default(openshift.docker.openshift_image_tag | default('')) }}"
+openshift_version: "{{ openshift_image_tag | default(openshift.docker.openshift_image_tag) | default('') }}"
```

Comment:
Why the switch from nested default filters? If openshift.docker.openshift_image_tag and openshift_image_tag are both undefined here you will get an error when evaluating openshift.docker.openshift_image_tag when applying the second default filter here.